js/auto_menu.js

```diff
@@ -1,30 +1,50 @@
 $( document ).ready(function() {
 
     var indexHtml = '';
-    indexHtml = indexHtml + '<ul>';
+    indexHtml = indexHtml + '<ul class="nav">';
     $( "h1" ).each(function() {
         if(typeof $( this ).attr('id') != 'undefined'){
 
-            indexHtml = indexHtml + '<li><a href="#' + $( this ).attr("id") + '"><b>' + $( this ).html() + '</b></a></li>';
+            indexHtml = indexHtml + '<li><a href="#' + $( this ).attr("id") + '"><b>' + $( this ).html() + '</b></a>';
 
-            indexHtml = indexHtml + '<ul>';
+            indexHtml = indexHtml + '<ul class="nav h2item">';
             $("h2[id^='" + $( this ).attr("id") + "_']").each(function() {
                 if(typeof $( this ).attr('id') != 'undefined'){
-                    indexHtml = indexHtml + '<li><a href="#' + $( this ).attr('id') + '">' + $( this ).html() + '</a></li>';
+                    indexHtml = indexHtml + '<li><a href="#' + $( this ).attr('id') + '">' + $( this ).html() + '</a>';
 
-                    indexHtml = indexHtml + '<ul>';
+                    indexHtml = indexHtml + '<ul class="nav h3item">';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/myparcelnl/magento/pullRequest?prid=8178215)
```suggestion
                    indexHtml = indexHtml + "<ul class=\"nav h3item\">";
```